bin/package_ios

```diff
@@ -24,8 +24,13 @@ PACKAGE_FILE="${DIR_BUILD}/Mysterium.framework.zip"
 DIR_TEMP=`mktemp -d ${DIR_BUILD}/${tempname}.XXXXXX`
 DIR_FRAMEWORK="${DIR_TEMP}/Mysterium.framework"
 
-xgo -image=mysteriumnetwork/xgo:1.11 -targets=ios/* -out Mysterium -dest $DIR_TEMP `pwd`/mobile/mysterium
+xgo \
+    -image=mysteriumnetwork/xgo:1.11 \
+    --ldflags="-w -s $(get_linker_ldflags)" \
+    -targets=ios/* -out Mysterium -dest $DIR_TEMP \
+    `pwd`/mobile/mysterium
 
+chown -R travis ${DIR_FRAMEWORK}
```

Comment:
This looks like a workaround for Travis CI only. What if I will run this script locally and I have no `travis` user?

Reply:
:(

Reply:
Well I guess we will find solution for local builds. Now fix that red master card